app/views/ratings/_rating.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= form_with(url: new_rating_path, class: "d-inline-block") do |rating| %>
```

Comment:
The block variable for `form_with` represents the form itself. You can name it whatever you want, but typically it's just called `form` or even `f` for short.

With the block variable you can [attach any form helper method](https://github.com/appdev-projects/helper-methods#forms-specifically-for-model-objects) to let Rails do even more work for you like pre-fill fields with values. More on that [here](https://uchicago.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=ada06fe0-0512-4667-9173-ad05013bda49&start=3342.52298). 

Essentially you can do things like

```erb
<%= form_with(model: Rating.new, class: "d-inline-block") do |form| %>
    <%= form.label :rating_value_box, "Your Rating" %>
    <%= form.number_field :rating_value, within: 0...6 %>
    ...
<% end %>
```